pkg/wallet/wallet.go

```diff
@@ -88,3 +152,19 @@ func (w *BaseWallet) persistKey(key *key) error {
 	}
 	return nil
 }
+
+// getKey get key
+func (w *BaseWallet) getKey(verkey string) (*key, error) {
```

Comment:
we should probably use the same key structure between our crypto and wallet packages (and remove the duplicate one)

Reply:
done